src/components/player/content/index.js

```diff
@@ -18,12 +22,29 @@ const Content = ({
   swap,
   toggleFullscreen,
 }) => {
+  const {
+    index,
+  } = useCurrentInterval(storage.tldraw);
+
   if (layout.single) return null;
 
   const isTldrawWhiteboard = storage.tldraw.length ||
                              storage.panzooms.tldraw ||
                              storage.cursor.tldraw;
 
+  let presentation = <Presentation />;
+  
+  if (isTldrawWhiteboard) {
+    const bbbVersion = getTldrawBbbVersion(index);
+
+    if (bbbVersion && !semverGte(bbbVersion, '3.0.0')) {
+      presentation = <TldrawPresentation />;
+    }
+    else if (bbbVersion) {
+      presentation = <TldrawPresentationV2 />;
+    }
+  }
+
```

Comment:
I recommend removing the negative from the logic, which should allow making the selection of whiteboard based on version a bit more clear. This also lets you make the `presentation` variable const.
```suggestion
  const presentation;
  
  if (isTldrawWhiteboard) {
    const bbbVersion = getTldrawBbbVersion(index);

    if (bbbVersion && semverGte(bbbVersion, '3.0.0')) {
      presentation = <TldrawPresentationV2 />;
    }
    else {
      presentation = <TldrawPresentation />;
    }
  }
  else {
    presentation = <Presentation />;
  }

```